src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedByDotComa = sourceString
```

Comment:
The variable name 'splittedByDotComa' is not descriptive of its contents. Consider renaming it to reflect that it contains style declarations split by semicolons.

Reply:
fixed. Now it's too long, but more clearer. I hope you're satisfied